code/modules/reagents/Chemistry-Recipes/Medicine.dm

```diff
@@ -57,6 +57,22 @@
 	result = "salglu_solution"
 	required_reagents = list("sodiumchloride" = 1, "water" = 1, "sugar" = 1)
 	result_amount = 3
+	
+/datum/chemical_reaction/mine_salve
+	name = "Miner's Salve"
+	id = "mine_salve"
+	result = "mine_salve"
+	required_reagents = list("oil" = 1, "water" = 1, "iron" = 1)
+	mix_message = "The fine black particles dissolve into the oily substance."
```

Comment:
Thematic niggle: these two messages (his, and the one in `mine_salve2`, below, are considerably more detailed than the other reaction messages. Keep in mind that these can be seen from several tiles away in some circumstances: does it make sense for someone at the chem door to be observing fine particles in anything?

I don't necessarily have an improved suggestion for you, but it stood out for me.